### PR TITLE
Add long description to setup.py and remove redundant dependencies

### PR DESCRIPTION
**Pull request by djhoese**
Fix #170

@qulogic I changed my mind 😄

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -108,10 +107,12 @@ def cythonize(extensions, **_ignore):
                 extension.sources[:] = sources
             return extensions
 
+    README = open('README.md', 'r').read()
     setup(name='pyresample',
           version=versioneer.get_version(),
           cmdclass=cmdclass,
           description='Resampling of remote sensing data in Python',
```
Comment (QuLogic) [inline setup.py]: Update description too?

File: setup.py [hunk 2]
```diff
@@ -108,10 +107,12 @@ def cythonize(extensions, **_ignore):
                 extension.sources[:] = sources
             return extensions
 
+    README = open('README.md', 'r').read()
```
Comment (QuLogic) [inline setup.py]: You'll need `long_description_content_type='text/markdown',` to get Markdown parsing on PyPI.

Comment (djhoese) [reply setup.py]: Thanks.

File: setup.py (hunk 1 above)
Comment (djhoese) [reply setup.py]: Done.

**Pull request merged**